README.md

```diff
@@ -4,6 +4,10 @@ Orbit's brand files, formats and guidelines.
 
 - [Guidelines](orbit_db_guidelines.pdf)
 
+In general, use the main standard logo when you can; the round one when you need a square logo of some sort; the hexagons for JavaScript related events (especially for stickers); and the wordless ("sans mot") logos when the icon needs to be small and the text isn't as important.
```

Comment:
"Round one when you need a square logo" is a little confusing. Otherwise, nice use of semicolons :)

Reply:
Well, it's when square or circular dimensions are called for. I'll clarify.